### PR TITLE
fix the little bug

### PR DESCRIPTION
**Pull request by ex860**

### CODE REVIEW

File: module/japanese_mix.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    print('<<'+word+'>>')
+    print(" ")
+
+    for i in jisho_Soup.find_all('div', class_='exact_block'):
+
+        firstBlock = i.find('div', class_='concept_light clearfix')
+        partJP = firstBlock.find('div', class_='concept_light-wrapper')
+        partEN = firstBlock.find('div', class_='concept_light-meanings')
+        status = partJP.find('div', class_='concept_light-status')
+        if(status != None):
```
Comment (hwchiu) [inline module/japanese_mix.py]: Make coding style consistent.
Use '()' or without it for all `if/else` statements,

File: module/japanese_mix.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                wget.download(source['src'], out=download_dir+"Jp_"+word+".mp3")
+                # Insert the sound media into the card
+                front_word += "[sound:Jp_"+word+".mp3]"
+        front_word += word + "<br>"
+        
+        for j in partJP.find_all('span', class_='furigana'):
+            furiCnt = 0
+            for child in j.children:
+                furiChild.append(child.string)
+                furiCnt = furiCnt + 1
```
Comment (hwchiu) [inline module/japanese_mix.py]: change to furiCnt++ ?

File: module/japanese_mix.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            furiList = list(filter(("\n").__ne__, furiChild))
+        for j in partJP.find_all('span', class_='text'):
+            textCnt = 0
+            for child in j.children:
+                textChild.append(child.string)
+                textCnt = textCnt + 1
```
Comment (hwchiu) [inline module/japanese_mix.py]: change to textCnt++ ?

File: module/japanese_mix.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            for k in range(0,len(textChild)):
+                for chr in _unicode_chr_splitter( textChild[k] ):
+                    if chr != '\n' and chr != ' ' and chr != '':
+                        textList.append(chr)
+        
+        for j in range(0,len(textList)):
+            if(furiList[j] == None):
```
Comment (hwchiu) [inline module/japanese_mix.py]: use '()' or without it for all `if/else` statement

File: module/japanese_mix.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+                reading += textList[j] 
+            else:
+                reading += " " + textList[j] + "[" + furiList[j] + "]" 
+        # for j in partEN.find_all('div', class_="meanings-wrapper"):
```
Comment (hwchiu) [inline module/japanese_mix.py]: remove those comment if we don't need than or add some comment to describe what we should keep them.

File: module/english_cambridge.py [hunk 6]
```diff
@@ -43,9 +43,9 @@ def LookUp(word, data):
             english = tabEntry.find('div', class_='tabs__content on', attrs={"data-tab": "ds-business-english"})
 
     partOfSpeech = english.find_all('div', class_='entry-body__el clrd js-share-holder')
-    sound = partOfSpeech[0].find('span', attrs={'data-src-mp3':True})['data-src-mp3']
+    sound = partOfSpeech[0].find('span', attrs={'data-src-mp3':True})
     if(sound is not None):
-        wget.download(sound, out=download_dir+"Py_"+word+".mp3")
+        wget.download(sound['data-src-mp3'], out=download_dir+"Py_"+word+".mp3")
```
Comment (hwchiu) [inline module/english_cambridge.py]: what will happen if the sound doesn't contains 'data-src-mp3' key ?

Comment (ex860) [reply module/english_cambridge.py]: I haven't faced this problem yet XD.
Maybe I can add one more expression to prevent this part of code from accident.

**Pull request merged**